tools/accuracy_checker/accuracy_checker/annotation_converters/ncf_converter.py

```diff
@@ -39,15 +39,15 @@ def parameters(cls):
         return parameters
 
     def configure(self):
-        self.rating_file = self.get_value_from_config('raiting_file')
+        self.rating_file = self.get_value_from_config('rating_file')
         self.negative_file = self.get_value_from_config('negative_file')
         self.users_max_number = self.get_value_from_config('users_max_number')
 
     def convert(self, check_content=False, **kwargs):
         annotations = []
         users = []
 
-        for file_row in enumerate(read_txt(self.rating_file)):
+        for index, file_row in enumerate(read_txt(self.rating_file)):
```

Comment:
`index` variable is not used, better to replace to underscore or replace cycle with `enumerate()` to usual iteration across 'read_txt(self.riting_file)'
I mean
```suggestion
        for file_row in read_txt(self.rating_file):
```